python/tutorials/06-fused-attention.py

```diff
@@ -497,38 +493,77 @@ def forward(ctx, q, k, v, causal, sm_scale):
 
     @staticmethod
     def backward(ctx, do):
+        # configuration is not supported
+        assert(not (ctx.split_kernel and not ctx.causal))
```

Comment:
I think bwd doesn't support causal in general so may be just `assert(ctx.causal)`?